buildSrc/settings.gradle.kts

```diff
@@ -29,3 +29,7 @@ dependencyResolutionManagement {
         }
     }
 }
+
+plugins {
+    id("org.gradle.toolchains.foojay-resolver-convention") version "0.7.0"
+}

```

Comment:
I am unable to download the specified JDK.

```
Failed to query the value of property 'buildFlowServiceProperty'.
> Could not isolate value org.jetbrains.kotlin.gradle.plugin.statistics.BuildFlowService$Parameters_Decorated@3ccc7b25 of type BuildFlowService.Parameters
   > A problem occurred configuring project ':buildSrc'.
      > Failed to calculate the value of task ':buildSrc:compileJava' property 'javaCompiler'.
         > No matching toolchains found for requested specification: {languageVersion=17, vendor=ADOPTIUM, implementation=vendor-specific} for LINUX on x86_64.
            > No locally installed toolchains match and toolchain download repositories have not been configured.
```

Reply:
Have you managed to solve it already?

Reply:
Yes, this change solves it.

Reply:
It's really interesting as I didn't have that error even without this change. I took another approach in https://github.com/NeoA11y/SpeakTouch/pull/131/commits/6257e6cdb2196447ee81b0bc1d4daef0bf4d4c69

Reply:
Ah, probably I understand now. You would like correct JDK to be downloaded while trying to build on another machine with e.g. JDK 21 installed.

Reply:
Yes, I realized it wasn't working properly because I took advantage of the recess to change my development setup. I installed open jdk and not temurin specifically, and then it threw an error during the build. As we specified the adoptium distro in the toolchain, even though the version was correct, since the distro was wrong, it wouldn't allow the build. But thanks to [Foojay Toolchains](https://github.com/gradle/foojay-toolchains), it should have been able to download and configure the correct distro on its own, which didn't happen.

From what I understand, since the toolchain was configured in buildSrc, Foojay Toolchains should have been too, which is why it didn't work.